### PR TITLE
Ember.js Test App

### PR DESCRIPTION
**Pull request by raghunath-r-a**
@QNNAKWUE Please verify that this is as expected, so that we can move with review/approval.

### CODE REVIEW

**Comment by QNNAKWUE** [comment]
I confirm this to be correct.


Best regards,

Queen Nnakwue

**Comment by raghunath-r-a** [comment]
@nathannguyenn Please approve this code PR. FYI, this is the one earlier identified as empty, but fixed now.

File: Emberjs/ember-auth/test-app/package.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    "eslint-plugin-ember": "^8.1.1",
+    "eslint-plugin-mocha": "^6.1.1",
+    "eslint-plugin-node": "^11.0.0",
+    "express": "^4.17.1",
+    "fastboot-app-server": "^2.0.0",
+    "glob": "^7.1.6",
+    "handlebars": "~4.7.1",
+    "loader.js": "^4.7.0",
+    "mocha": "^7.0.1",
+    "my-engine": "./packages/test-app/lib/my-engine",
```
Comment (nathannguyenn) [inline Emberjs/ember-auth/test-app/package.json:64]: the path for `"my-engine"` here is incorrect and will cause an error when doing `yarn install` - correct path is `"./lib/my-engine"`

**Review by nathannguyenn — CHANGES_REQUESTED** [review]
1 issue highlighted, once fixed it can be merge, everything else looks good!

**Pull request merged**